fili-core/src/main/java/com/yahoo/bard/webservice/table/ConcretePhysicalTable.java

```diff
@@ -70,7 +75,6 @@ public String getFactTableName() {
 
     @Override
     public String toString() {
-        return super.toString() + " factTableName: " + getAvailability().getDataSourceNames() + " alignment: " +
-                getTableAlignment();
+        return super.toString() + " factTableName: " + getAvailability().getDataSourceNames();
```

Comment:
Thanks for hunting down what's happening. I'm OK not tackling it right now, but we should add a techdebt issue that explains the change we made, why we made it, and what we should do to fix it.

To fix this, we should put the `getTableAlignment()` back into the toString for ConcretePhysicalTable, and then fix up the ClassScanner so that it's able to build a valid instance of all of the things in that chain you walked through so that it works.